src/org/usfirst/frc/team694/robot/subsystems/Lift.java

```diff
@@ -97,14 +95,18 @@ public boolean isAtTop() {
     }
 
     public double getLeftLiftEncoderDistance() {
-        return leftLiftMotor.getSelectedSensorPosition(0) * RobotMap.LIFT_RAW_MULTIPLIER;
+        return leftLiftMotor.getSelectedSensorPosition(0) * RobotMap.LIFT_ENCODER_RAW_MULTIPLIER;
     }
 
     public double getRightLiftEncoderDistance() {
-        return rightLiftMotor.getSelectedSensorPosition(0) * RobotMap.LIFT_RAW_MULTIPLIER;
+        return rightLiftMotor.getSelectedSensorPosition(0) * RobotMap.LIFT_ENCODER_RAW_MULTIPLIER;
     }
 
     public double getMaxLiftEncoderDistance() {
         return Math.max(getLeftLiftEncoderDistance(), getRightLiftEncoderDistance());
     }
+    
+    public double getLiftHeight() {
+        return getMaxLiftEncoderDistance() + RobotMap.MIN_HEIGHT_OF_GRABBER;
```

Comment:
This works out ok, but I'd consider where "getLiftHeight" is used. Do we care how high this is off of the ground? If so, then keep your changes. Otherwise, we may only care about how far the lift has traveled.

For now, I'd suggest keeping your changes, because it works out for now.